src/material-experimental/mdc-slider/_slider-theme.scss

```diff
@@ -1,22 +1,43 @@
-// TODO: disabled until we implement the new MDC slider.
-// @import '@material/slider/mixins.import';
+@import '@material/slider/slider';
+@import '@material/slider/slider-theme';
 @import '../mdc-helpers/mdc-helpers';
+@import '../../material/core/ripple/ripple';
 
 @mixin mat-mdc-slider-color($config-or-theme) {
   $config: mat-get-color-config($config-or-theme);
   @include mat-using-mdc-theme($config) {
-    // TODO: disabled until we implement the new MDC slider.
-    // @include mdc-slider-core-styles($query: $mat-theme-styles-query);
+    @include without-ripple($query: $mat-theme-styles-query);
 
     .mat-mdc-slider {
+      &.mat-primary, &.mat-accent, &.mat-warn {
+        @include value-indicator-color(
+          $color: #000,
```

Comment:
Should this be hardcoded to black? it feels like it might not work with dark themes

Reply:
What would be the appropriate var for this? `on-surface`?

For context, the value-indicator-color is the color of the tooltip that appears above discrete slider thumbs

Reply:
What does this color wind up getting applied to? is it a text color or a background color?

Reply:
<img width="92" alt="Screen Shot 2021-02-18 at 10 26 55 AM" src="https://user-images.githubusercontent.com/25158423/108403788-f6590780-71d3-11eb-9b2c-3eb2f45e50a9.png">
The `value-indicator-color` is the background and the `value-indicator-text-color` is the text

Reply:
It looks like MDC doesn't do a very good job of handling this: https://github.com/material-components/material-components-web/blob/master/packages/mdc-slider/_slider-theme.scss#L36-L38

They assume that black will look good on the background and that the on-primary text color will be white. I don't think either of those things is guaranteed. Instead I would use our theme config to choose the appropriate color:

```scss
$is-dark: map-get($config, is-dark);
$indicator-color: if($is-dark, white, black);
$indicator-text-color: if($is-dark, black, white);
```

Reply:
Great catch here. I think it looks much better 👍🏽 

Here's a before and after of the slider in dark mode

Before:
<img width="967" alt="Screen Shot 2021-02-18 at 11 53 37 AM" src="https://user-images.githubusercontent.com/25158423/108413543-00810300-71e0-11eb-9b48-60cb68909448.png">

After:
<img width="968" alt="Screen Shot 2021-02-18 at 11 48 20 AM" src="https://user-images.githubusercontent.com/25158423/108413372-cb74b080-71df-11eb-987f-dcaccdb04ff6.png">

Reply:
You might want to bump the opacity a bit in dark mode. It looks much better, but still kinda faded (I couldn't find anything in the spec that mentioned what it should be for dark mode though)